src/materialized/ci-slim/Dockerfile

```diff
@@ -20,4 +20,4 @@ COPY materialized /usr/local/bin/
 
 USER materialize
 
-ENTRYPOINT ["tini", "--", "materialized", "--listen-addr=0.0.0.0:6875"]
+ENTRYPOINT ["tini", "--", "materialized", "--sql-listen-addr=0.0.0.0:6875", "--http-listen-addr=0.0.0.0:6875"]
```

Comment:
Does `--http-listen-addr` need to be `6876` here?

Reply:
Good catch. I have no idea why this did not result in a test failure.